shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm

```diff
@@ -32,6 +35,20 @@
 
 static const char* kApplicationKernelSnapshotFileName = "kernel_blob.bin";
 
+static BOOL DoesHardwareSupportsWideGamut() {
+  id<MTLDevice> device = MTLCreateSystemDefaultDevice();
+  BOOL result;
+  if (@available(iOS 13.0, *)) {
+    // MTLGPUFamilyApple2 = A9/A10
+    result = [device supportsFamily:MTLGPUFamilyApple2];
+  } else {
+    // A9/A10 on iOS 10+
+    result = [device supportsFeatureSet:MTLFeatureSet_iOS_GPUFamily3_v2];
```

Comment:
These gates were selected by looking at the hardware/os of the iPhone 7 which was the first device to support wide gamut.

Reply:
Sure, done.